samples/crypto/rsa/src/main.c

```diff
@@ -146,11 +173,11 @@ int sign_message_rsa(void)
 			       sizeof(m_signature),
 			       &olen);
 	if (status != PSA_SUCCESS) {
-		PRINT_ERROR("psa_sign_hash", status);
+		LOG_INF("psa_sign_hash failed! (Error: %d)", status);
 		return APP_ERROR;
 	}
 
-	PRINT_MESSAGE("Singing was successful!");
+	LOG_INF("Singing was successful!");
```

Comment:
Even though it currently is a lot cuter, I guess "signing" is what the sample is doing at this stage😄

Reply:
Hahaha :D I am actually so tempted to leave it like this but I will update it.